web/registry/models.py

```diff
@@ -9,64 +9,54 @@ class Meta:
         abstract = True
 
 
+class ComponentDependency(TimeStampedModel):
+    depender = models.ForeignKey(
+        "Component", on_delete=models.CASCADE, related_name="depender_set"
+    )
+    dependee = models.ForeignKey(
+        "Component", on_delete=models.CASCADE, related_name="dependee_set"
+    )
+    alias = models.TextField()
```

Comment:
You don't necessarily have to change this since new users of Components probably won't be looking at this code anyway, but in general I feel like calling this "attribute_name" instead of "alias" would introduce one fewer new concepts to developer learning component programming for the first time, since they will already know what Python attributes are.